components/table/test/table.vdiff.js

```diff
@@ -0,0 +1,407 @@
+import '../../inputs/input-number.js';
+import '../demo/table-test.js';
+import '../table-col-sort-button.js';
+import { defineCE, expect, fixture, focusElem, html, nextFrame } from '@brightspace-ui/testing';
+import { LitElement, nothing } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { tableStyles } from '../table-wrapper.js';
+
+function createHeaderRow(opts) {
+	const { headerAttribute, trClass } = { headerAttribute: false, ...opts };
+	return html`
+		<tr class="${ifDefined(trClass)}" ?header="${headerAttribute}">
+			<th>Header A</th>
+			<th>Header B</th>
+			<th>Header C</th>
+		</tr>
+	`;
+}
+function createSortableHeaderRow() {
+	return html`
+		<tr>
+			<th><d2l-table-col-sort-button>Ascending</d2l-table-col-sort-button></th>
+			<th><d2l-table-col-sort-button desc>Descending</d2l-table-col-sort-button></th>
+			<th><d2l-table-col-sort-button nosort>No Sort</d2l-table-col-sort-button></th>
+		</tr>
+	`;
+}
+function createFruitHeaderRows(opts) {
+	const { selectable, headerAttribute, stickyAttribute, stickyClass, trClass } = { selectable: false, headerAttribute: false, stickyAttribute: false, ...opts };
+	return html`
+		<tr class="${ifDefined(trClass)}" ?header="${headerAttribute}">
+			${selectable ? html`<th rowspan="2" style="width: 1%" class="top"><input type="checkbox"></th>` : nothing}
+			<th class="${ifDefined(!selectable ? 'top' : stickyClass)}" ?sticky="${stickyAttribute}" rowspan="2">Country</th>
+			<th colspan="5">Fruit Production (tons)</th>
+		</tr>
+		<tr class="${ifDefined(trClass)}" ?header="${headerAttribute}">
+			<th>Apples</th>
+			<th>Oranges</th>
+			<th>Bananas</th>
+			<th>Peaches</th>
+			<th>Grapes</th>
+		</tr>
+	`;
+}
+
+function createRows(keys, opts) {
+	const { selected } = { selected: false, ...opts };
+	return html`${keys.map(key => html`
+		<tr ?selected="${selected}">
+			<td>${`Cell ${key}-A`}</td>
+			<td>${`Cell ${key}-B`}</td>
+			<td>${`Cell ${key}-C`}</td>
+		</tr>
+	`)}`;
+}
+function createFruitRows(opts) {
```

Comment:
This function does a lot of things that aren't used yet, but will be used by the `sticky` tests